idseq_dag/engine/pipeline_step.py

```diff
@@ -262,3 +266,27 @@ def step_resources(self):
         By default, show link to idseq-dag documentation.
         '''
         return {"IDseq Docs": "https://github.com/chanzuckerberg/idseq-dag/wiki"}
+
+
+class PipelineCountingStep(PipelineStep):
+
+    def input_cluster_sizes(self):
```

Comment:
nit: I wouldn't expect such a function name to return a file path. How about `input_cluster_sizes_path` ?

Reply:
sounds good